Sources/Sentry/Profiling/SentryLaunchProfiling.m

```diff
@@ -39,7 +39,8 @@
     SentryTracerConfiguration *config = [SentryTracerConfiguration defaultConfiguration];
     config.profilesSamplerDecision =
         [[SentrySamplerDecision alloc] initWithDecision:kSentrySampleDecisionYes
-                                          forSampleRate:profilesRate];
+                                          forSampleRate:profilesRate
+                                         withSampleRand:@0.9999];
```

Comment:
Not sure if this is valid, the decision is hard-coded to `YES`, so I would go with `1.0` sample rand, but the docs mention the range to be `[0,1)` excluding `1.0`